crates/broomva-cli/install.sh

```
@@ -5,6 +5,46 @@ REPO="broomva/broomva.tech"
+# The Broomva Stack is a living roster. Rather than hardcode a count that
+# drifts every time we add or rename a skill, we fetch the authoritative
+# source — the ROSTER array in bstack's own SKILL.md — at install time.
+#
+# Falls back gracefully to a generic phrasing if the network is unavailable
+# or the bstack repo's SKILL.md layout changes.
+BSTACK_SKILL_MD_URL="${BSTACK_SKILL_MD_URL:-https://raw.githubusercontent.com/broomva/bstack/main/SKILL.md}"
+
+get_bstack_skill_count() {
+  # Pull bstack/SKILL.md, extract the ROSTER=( … ) array, count entries.
+  # ROSTER tokens are bare lowercase identifiers separated by whitespace.
+  local count
```

Shellcheck (0.11.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)